web/app/js/components/Endpoints.jsx

```diff
@@ -29,11 +31,13 @@ const endpointColumns = [
   {
     title: "Pod",
     dataIndex: "name",
-    sorter: (a, b) => (a.name).localeCompare(b.name)
+    sorter: (a, b) => (a.name).localeCompare(b.name),
+    render: d => <Link to={`/namespaces/${d.namespace}/pods/${d.name}`}>{d.name}</Link>
   },
   {
     title: "Resource Version",
-    dataIndex: "pod.resourceVersion"
+    dataIndex: "pod.resourceVersion",
+    sorter: (a, b) => (a.pod.resourceVersion).localeCompare(b.resourceVersion)
```

Comment:
are these numbers stored as strings? will this sort numerically?

<img width="965" alt="screen shot 2019-02-13 at 4 48 42 pm" src="https://user-images.githubusercontent.com/236915/52754405-3d8ddb80-2faf-11e9-8574-90177a741950.png">

Reply:
yeah they are, but maybe I should set `numeric: true` on the `localeCompare` so that they are sorted as numbers